indico/modules/events/abstracts/util.py

```diff
@@ -379,3 +382,15 @@ def get_events_with_abstract_persons(user, dt=None):
     for person in query:
         data[person.event_id].add('abstract_person')
     return data
+
+
+def filter_field_values(fields, user, managers, submitter):
+    if user in managers:
+        return {field for field in fields if field.contribution_field.is_active}
+    if user == submitter:
```

Comment:
I think we want the `abstract.user_owns()` check here and not an explicit check for the submitter.